serverless.example.yml

```diff
@@ -49,6 +49,17 @@ functions:
       - http:
           path: beacons/on_chain_value
           method: get
+          # Functions often require an associated CORS OPTIONS method enabled on the API gateway to satisfy Chrome pre-flight
+          # requests. An OPTIONS method is generally not required for GET requests but is always required by Chrome for
```

Comment:
This comment is not true. CORS is not required for certain POST requests. I'd just comment this:

```
Enable CORS for every endpoint. Based on https://www.serverless.com/blog/cors-api-gateway-survival-guide/
```

Reply:
Ah I see, yes, it's more complex than just the POST method. Thanks for correcting me.